samples/dotnet/clientcreds/Lenus.Samples.ClientCredentialsFlow/src/Configuration/LenusClientOptions.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Lenus.Samples.ClientCredentialsFlow.Configuration
+{
+	public class LenusClientOptions
+	{
+        public string TokenUrl { get; set; } = string.Empty;
+        public string ClientId { get; set; } = string.Empty;
+        public string ClientSecret { get; set; } = string.Empty;
+    }
```

Comment:
Again just formatting, there's some odd whitespace going on in this file.

Reply:
Fixed the whitespace issue